prisma/production-seed/users.ts

```diff
@@ -0,0 +1,358 @@
+import { prisma } from "../seed/prisma-client";
+import { hashPassword } from "../../src/utils/auth";
+import { getRoleId } from "../seed/users";
+
+export const populateUsersProd = async () => {
+    const roles = await prisma.role.findMany({});
+
+    //<editor-fold desc="admin account">
+    const adminUser = await prisma.user.create({
+        data: {
+            email: "admin@example.com",
+            password: await hashPassword("password"),
+            emailVerified: true,
+            firstName: "Admin",
+            lastName: "Surname",
+            avatar: "https://gravatar.com/avatar/3bfaef00e02a22f99e17c66e7a9fdd31?s=400&d=robohash&r=x",
```

Comment:
I think if you want different generated robot, you have to change the hash for each user. Also I don't think it means anything but `r=x` is the gravatar images rating. We could probably just remove the rating, because the default rating is `g`, which allows for only g-rated images to be displayed on the website. 
[avatars request - gravatar docs](https://docs.gravatar.com/api/avatars/images/)